views/use_the_data.erb

```diff
@@ -0,0 +1,256 @@
+			Balance will incorporate the change it as soon as EveryPolitician has the
+			data.
+		</p>
+		<p>
+			You need to understand GitHub&rsquo;s webhooks to do this, but in a
+			nutshell you tell GitHub to make a request to a URL that you&rsquo;ve
+			set up to react when a particular event occurs (such as the 
+      <code>everypolitician-data</code> repo merging a pull request, which is how
+      we manage the data).
+		</p>
+		
+		<h2>
+			Pick a format
+		</h2>
+		<p>
+			The data itself is available in two formats. Which one you want will
+			depend on what you&rsquo;re trying to do with it.
+		</p>
+    <ul>
+      <li><a href="#csv-data">CSV data</a> for straightforward data for a given term</li>
+      <li><a href="#json-data">Popolo JSON data</a> for full structured data</li>
+    </ul>
+
+    <a name="csv-data"></a>
+		<h3>
+			CSV data
+		</h3>
+		<p>
+			We make the data available as CSV (comma separated values) because
+			it&rsquo;s just so <em>useful</em>. If you just want the basic data, it
+			might also be the simplest way for you to absorb it. For any given
+			legislature we slice the data up into separate CSV files for each term.
+			("Terms" are important: we think, for example, that the most common use of
+			the data will be applications using the data for the current term, that is,
+			data about who&rsquo;s in power now).
+		</p>
+		<p>
+			The disadvantage of the CSV format is it can&rsquo;t be as rich,
+			because, by definition, it can&rsquo;t easily represent structure. So
+			that&rsquo;s what the JSON is for.
+		</p>
+    <a name="json-data"></a>
+		<h3>
+			Popolo JSON data
+		</h3>
+		<p>
+			Popolo is an open standard for expressing political data &mdash; exactly
+			for the kind of thing we&rsquo;re doing here. So we provide our data in JSON
+			format too, complying with Popolo standard. These are the same politicians
+			as in the CSV of course (we generate both the CSV and JSON anew every time
+			anything changes, keeping them perfectly in synch), but the data is richer.
+			Because Popolo can express such things well, the JSON data spans terms
+			&mdash; it has membership dates within it. So, unlike the CSV which is
+			sliced by terms, if you pull down the JSON data you&rsquo;ll be getting
+			<em>all</em> the data for all the politicians across all the terms.
+		</p>
+		<p>
+			If you&rsquo;re doing anything which needs this sort of richness,
+			grab the JSON data and use your favourite JSON library to jump right into
+			having structured data in your application.
+		</p>
+    <a name="#rawgit"></a>
+		<h2>
+			Where to get it from: <code>rawgit.com</code>
+		</h2>
+		<p>
+			Although the EveryPolitician data is published on GitHub, your code
+			should&rsquo;t try to download it from there. You can get away with that as
+			a human, but if your application tries it, it will run into problems.
+		</p>
+		<p>
+			Because we &ldquo;publish&rdquo; our data on GitHub, it&rsquo;s
```

Comment:
Removed; this was based on me earlier saying "get it from rawgit.com" so in that sense it doesn't feel like publishing is the right word for putting it on github; but that's me being overcautious about the use of rawgit -- as you say, that's really for the specific case of client consumption, so it's not unreasonable for someone to get data from github, if that works for them, right?